test/distributed/test_c10d_common.py

```diff
@@ -806,6 +806,28 @@ def fut_then(fut):
 
         return fut.then(fut_then)
 
+    @skip_if_lt_x_gpu(2)
+    def test_sync_batch_norm_empty_input(self):
+        pg = self._get_process_group()
+
+        model = torch.nn.Sequential(
+            torch.nn.BatchNorm2d(2),
+        ).to(device=self.rank)
+        model = DistributedDataParallel(
+            model,
+            device_ids=[self.rank],
+            process_group=pg,
+        )
+        model = torch.nn.SyncBatchNorm.convert_sync_batchnorm(
+            model,
+            process_group=pg,
+        )
+
+        model.train()
+        x = torch.zeros((1 if self.rank != 0 else 0, 2, 11, 13), dtype=torch.float32, device=self.rank)
```

Comment:
We are passing an input with a single value here (all zeroes) which means that the standard deviation will be 0 so it's inverse would be `inf` which might be the source of your `nan`. 

Perhaps we should change this to
```suggestion
        x = torch.randn((1 if self.rank != 0 else 0, 2, 11, 13), dtype=torch.float32, device=self.rank)
```